### DISCUSSION TITLE
How do I "merge" all my local repo and overwrite what's shown on github?

### DISCUSSION THREAD

**Discussion by enjoysmath** [community/community#23011] [New to GitHub]
<p><a href="https://github.com/enjoysmath/BananaCats" rel="nofollow noopener">https://github.com/enjoysmath/BananaCats</a></p>
<p>Is the repo.  That is waaaayyy old code.  I need to merge in my current code from my local repo, using TortoiseGit.  I’ve tried “merge”.  I am pushing to master just fine.</p>

**Comment by enjoysmath** [comment]
<p>Any ideas on this?</p>

**Comment by enjoysmath** [comment, marked as answer]
<p>Solved this, by cloning what’s on <a href="http://github.com" rel="nofollow noopener">github.com</a>, deleting everything locally, copy/pasting in from my local folder all the new stuff, comming and pushing.  Now the changes are there.</p>
<p>Might have been that I was pushing to bitbucket, or something…</p>

**Comment by database007** [comment]
<p>I have created a pull request successfully, but couldnt ‘merge request’ and confirm… Need help on this ASAP… its urgent</p>
